specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -2939,10 +2939,10 @@
         },
         "crv": {
```

Comment:
While you are at it consider making a common reference instead of duplicating here.